r2/r2/lib/importer.py

```diff
@@ -105,7 +105,7 @@ def process_post(self, post_data, sr):
 
         if not post:
             # Create new post
-            post = Link._submit(title, article, account, sr, ip, tags, date=utc_date)
+            post = Link._submit(title, article, 'self', account, sr, ip, tags, date=utc_date)
```

Comment:
I'm fairly certain these other two are dead code, but I added the parameter anyways.